### PR TITLE
Update from upstream

### PR DESCRIPTION
**Pull request by architectbot**
This PR has been created from automation in https://github.com/giantswarm/grafana-helm-charts-upstream

### CODE REVIEW

**Comment by QuentinBisson** [comment]
Tested on Gauss

File: helm/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,14 +1,16 @@
 apiVersion: v2
 name: grafana
-version: 0.4.0
-appVersion: 8.4.2
```
Comment (TheoBrigitte) [inline helm/grafana/Chart.yaml:4]: That's a major bump. Any breaking changes?

I would like to see changelog somewhere or at least important updates and a major version bump for `version`

**Review by TheoBrigitte — CHANGES_REQUESTED** [review]
What was changed by us and what came from upstream ?

File: helm/grafana/Chart.yaml (hunk 1 above)
Comment (QuentinBisson) [reply helm/grafana/Chart.yaml:4]: No breaking changes no as Grafana mostly keeps compatibility hence I did not see the need to bump the major version but I can do it

Comment (QuentinBisson) [reply helm/grafana/Chart.yaml:4]: https://grafana.com/docs/grafana/latest/release-notes/release-notes-9-0-0/#breaking-changes

**Review by hervenicol — CHANGES_REQUESTED** [review]
These upstream updates are really hard to read, there's a lot of changes.
So I assume the upstream chart works, no need to review all the stuff that comes from upstream.

Now I'd like to have a (few) line(s) in CHANGELOG listing the components upgrades, and if I should expect to break things or not when deploying, or if I should care about some specific config in my values.yaml.

File: helm/grafana/Chart.yaml (hunk 1 above)
Comment (QuentinBisson) [reply helm/grafana/Chart.yaml:4]: The things changed by us are merely disabling pspUseAppArmor and enabling psp

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Upgrade to version 6.31.0 of the upstream chart with grafana 9.0.1.
```
Comment (hervenicol) [inline CHANGELOG.md:12]: I prefer when it says "**from** xxx to xxx", especially when there's a gap, it helps reading the upstream changelog for breaking changes.

Comment (QuentinBisson) [reply CHANGELOG.md:12]: :ack:

**Comment by QuentinBisson** [comment]
Can you check the changelog again ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Upgrade from version 6.24.1 to 6.31.0 of the upstream chart from grafana 8.4.2 to grafana 9.0.1. This release includes a small set of breaking changes that you can check [here](https://grafana.com/docs/grafana/latest/release-notes/release-notes-9-0-0/#breaking-changes).
```
Comment (hervenicol) [inline CHANGELOG.md:12]:
```suggestion
- Upgrade upstream chart from version 6.24.1 to 6.31.0, and grafana from 8.4.2 to 9.0.1. This release includes a small set of breaking changes that you can check [here](https://grafana.com/docs/grafana/latest/release-notes/release-notes-9-0-0/#breaking-changes).
```

**Pull request merged**